test/src/test/java/org/corfudb/runtime/LogReplicationServiceClientTest.java

```diff
@@ -0,0 +1,380 @@
+package org.corfudb.runtime;
+
+import com.google.common.collect.Iterables;
+import org.corfudb.integration.AbstractIT;
+import org.corfudb.runtime.LogReplication.ClientDestinationInfoKey;
+import org.corfudb.runtime.LogReplication.ClientRegistrationId;
+import org.corfudb.runtime.LogReplication.ClientRegistrationInfo;
+import org.corfudb.runtime.LogReplication.DestinationInfoVal;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.CorfuStoreEntry;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableOptions;
+import org.corfudb.test.SampleSchema;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LogReplicationServiceClientTest extends AbstractIT {
```

Comment:
Instead of an IT, unit tests for the different apis of the client will be more lightweight